src/ReleaseHistory.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## Unreleased
+
+- FEATURE: Users are able to opt-in for additional checks before submitting a SARIF file. [#139](https://github.com/microsoft/sarif-website/pull/139)
```

Comment:
> FEATURE: Users are able to opt-in for additional checks before submitting a SARIF file. [#139](https://github.com/microsoft/sarif-website/pull/139) [](http://example.com/codeflow?start=1&length=148)

BUG: Users now can select the 'additional suggestsions' and 'github ingestion rules' checkboxes in advance of validation. Previously, these additional checks would have been run in any case.